### PR TITLE
Add the possibility to run without visual odometry

### PR DESCRIPTION
**Pull request by dogoepp**
This would enable us to make demos with pre-recorded controllers without the need for the unstable visual odometry.

### CODE REVIEW

**Comment by costashatz** [comment]
Looks good. Did you try it on the real robot?

File: config/hexa_control_odomless.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Serial
+SerialPort: /dev/ttyACM0
+
+# Baudrate: 1 - 1000000, 16 - 115200, 34 - 57600
+SerialBaudrate: 1
+
+# Odometry Topic
+OdomEnable: false
```
Comment (jbmouret) [inline config/hexa_control_odomless.yaml:8]: I vote we enable odometry by default, as I can imagine myself searching for hours why the odometry is not activated / learning does not happen as expected.

Comment (costashatz) [inline config/hexa_control_odomless.yaml:8]: Right now we have many config files (one for each case). This is bad, but we discussed it with Dorian and when I have the `hexapod_ros` repo, the `resibots_launch` will be deprecated (deleted). So, I think for now is ok and we should discuss how to do it properly in the next meeting.

Comment (jbmouret) [inline config/hexa_control_odomless.yaml:8]: OK. I did not see it was a config file specific to the 'odomless' case.

**Comment by dogoepp** [comment]
I did try on the real Pexod:
- without visual odometry and using hexapod_client
- with the odometry and using limbo

I'll try the mixed cases to be sure it doesn't crash.

**Comment by jbmouret** [comment]
With the small power supply?

**Comment by dogoepp** [comment]
Sure, with the small power supply. It does not seem to hinder the robot 
until now.

**Comment by dogoepp** [comment]
All goes smoothly with limbo and hexapod_client, whenever the odometry is enabled or disabled.

Note that when it is disabled, the client (limbo or hexapod_client) gets a travelled distance of -1.

Note also that I could make two runs of limbo code (learning) with only the small power supply.

**Comment by dogoepp** [comment]
OK for merging ?

**Comment by costashatz** [comment]
**OK for me**

**Comment by jbmouret** [comment]
**OK for me**
On 08 Dec 2015, at 10:01, Konstantinos Chatzilygeroudis notifications@github.com wrote:

> OK for me
> 
> �
> Reply to this email directly or view it on GitHub.

� Jean-Baptiste Mouret
http://pages.isir.upmc.fr/~mouret/

**Pull request merged**